NorwegianRefugeeCouncil/core
# Task/core24 120 me api endpoint

Jira ticket: JIRA-CORE24-120

## What have you implemented?

* Add /healthz endpoint that checks db connection and returns 200. Does not require auth.
* Move api router to a controller file
* Add me router with endpoint to get user data that was preloaded into the request
* Delete welcome route
* Add seeding for users in oidc-users.json

## How to test it?
* Go to https://localhost:10000/api/me
* Log in as admin user
* See the json response of the user

## Extra/next steps
* Deploy and test
